client/my-sites/plugins/plugin.jsx

```diff
@@ -385,6 +386,7 @@ const SinglePlugin = React.createClass( {
 
 		return (
 			<MainComponent>
+				<QuerySites allSites />
```

Comment:
This was the quickest win I could find to make sure we have sites in the redux store.
Without sites in the store we can't use the `isSiteAutomatedTransfer` selector